### ISSUE TITLE
Add hover effect to entire card if a link is present

### ISSUE THREAD

**Issue by bspeare** [uiowa/uids#460]
# Story

As a site user, I can hover over a card and see a visual indication that the card contains a link so that I can easily distinguish between multiple linked cards. 

# Proposed Solution

- Add animated underline effect in use on the "Areas of Study" banners on https://viewbook.admissions.uiowa.edu.
- Or something more fancy like https://admission.princeton.edu/academics/faculty-profiles

**Comment by mjoneill** [comment]
@bspeare can you add clarification to this? Why this is important?

**Comment by bspeare** [comment]
@mjoneill the intention is to give a visual indication on hover that the card contains a link so that it can easily distinguished between multiple linked cards.  Admissions requested this as a "must-have" feature.

**Comment by joewhitsitt** [comment]
This was brought up in a discussion about https://github.com/uiowa/uids/issues/553, "Card links accept CTRL+Click to open in new window?"

Hover/focus would be a nice addition, but still won’t solve the right-click open new tab if the click doesn't happen on the card title correct?

**Comment by bspeare** [comment]
@joewhitsitt That's correct.   We could remove the hover/focus styles on the faux button and create a better hover/focus style on the card title to draw attention to that as the primary link source.
